index.html

```diff
@@ -157,7 +158,19 @@
 
       sheet
         .filter(row => row.business_name != 'Business Name' && !!row.business_name)
-        .sort((a, b) => {return 0.5 - Math.random()})
```

Comment:
Nice. Let's actually do something like the following.

```javascript
      const sortAlpha = (a, b) => {
        if (a.business_name.toLowerCase() < b.business_name.toLowerCase()) {
          return -1;
        }
        if (a.business_name.toLowerCase() > b.business_name.toLowerCase()) {
          return 1;
        }
        return 0;
      };
      const sortRandom = (a, b) => {
        return 0.5 - Math.random();
      };
      const sortFunc = RANDOMIZE_BUSINESS_LIST ? sortRandom : sortAlpha;

      sheet
        .filter(row => row.business_name != 'Business Name' && !!row.business_name)
        .sort(sortFunc)
        .map((row, i) => {
```

Reply:
makes sense — made the appropriate changes